ACPs/103-dynamic-fees/README.md

```diff
@@ -0,0 +1,135 @@
+```text
+ACP: 103
+Title: Add Dynamic Fees to the X-Chain and P-Chain
+Author(s): Dhruba Basu (https://github.com/dhrubabasu), Alberto Benegiamo (https://github.com/abi87), Stephen Buttolph (https://github.com/StephenButtolph)
+Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Introduce a dynamic fee mechanism to the X-Chain and the P-Chain. Preview a future transition to a multidimensional fee mechanism.
+
+## Motivation
+
+Under a fixed fee mechanism, there is a greater likelihood of higher network congestion. During periods of high demand, there is no economic incentive for users to withhold issuance of their transactions, extending the period of high demand.
+
+The C-Chain, in [Apricot Phase 3](https://medium.com/avalancheavax/apricot-phase-three-c-chain-dynamic-fees-432d32d67b60), employs a dynamic fee mechanism to raise the price during periods of high demand and lowering the price during periods of low demand. As the price gets too expensive, network utilization will decrease, which drops the price.
+
+The X-Chain and P-Chain currently operate under a fixed fee mechanism. To better handle increased load, they should be migrated to a dynamic fee mechanism. This migration should be done in tandem with [ACP-77](../77-reinventing-subnets/README.md) to ensure developers can account for the dynamic fee mechanism.
+
+## Specification
+
+### Dimensions
+
+There are four dimensions that will be used to approximate the computational cost ("gas" usage) of a transaction:
+
+1. `Bandwidth` ($B$) is the amount of network bandwidth used in transaction broadcast. This is set to the size of the transaction in bytes.
+2. `Reads` ($R$) is the number of state/database reads used in transaction execution.
+3. `Writes` ($W$) is the number of state/database writes used in transaction execution.
+4. `Compute` ($C$) is the total amount of compute used in transaction execution. This is set to the number of signatures verified during transaction execution.
```

Comment:
Is this going to explicitly be the number of signatures? Or do we just thing that sig verification will dominate this value?
```suggestion
4. `Compute` ($C$) is the total amount of compute used in transaction execution.
```

Reply:
adjusted the wording